Gradle task downloadNativeDependencies downloads to wrong folder

- This task copies native dependencies into `/build/resources/native` whereas the task `gradle bootRun` expects them to be in `/lib`. This means that in an IDE (except IntelliJ?) or when only using gradle without IDE, you need to copy around the native files manually before being able to run the client.

- This task should be a dependency of any task that runs the client (like `bootRun`) so that it doesn't need to be executed manually. (DONE)

This is still an issue. Eclipse example (but running gradle from cmd will run into the same problem):
<details>
<summary>Expand</summary>
<p>

```
com.github.nocatch.NoCatchException: java.io.IOException: Cannot run program "lib/faf-uid.exe": CreateProcess error=2, The system cannot find the file specified
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at com.github.nocatch.NoCatch.wrapException(NoCatch.java:61)
	at com.github.nocatch.NoCatch.noCatch(NoCatch.java:104)
	at com.github.nocatch.NoCatch.noCatch(NoCatch.java:82)
	at com.faforever.client.remote.FafServerAccessorImpl.lambda$connectAndLogIn$5(FafServerAccessorImpl.java:126)
	at com.faforever.commons.lobby.FafLobbyClient$handle$1.call(FafLobbyClient.kt:169)
	at com.faforever.commons.lobby.FafLobbyClient$handle$1.call(FafLobbyClient.kt:164)
	at reactor.core.publisher.MonoCallable.subscribe(MonoCallable.java:56)
	at reactor.core.publisher.Mono.subscribe(Mono.java:3987)
	at reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:425)
	at reactor.core.publisher.FluxFlatMap$FlatMapMain.tryEmit(FluxFlatMap.java:542)
	at reactor.core.publisher.FluxFlatMap$FlatMapInner.onNext(FluxFlatMap.java:1006)
	at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onNext(FluxOnErrorResume.java:79)
	at reactor.core.publisher.Operators$MonoSubscriber.complete(Operators.java:1784)
	at reactor.core.publisher.MonoCallable.subscribe(MonoCallable.java:61)
	at reactor.core.publisher.Mono.subscribe(Mono.java:3987)
	at reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:425)
	at reactor.core.publisher.FluxPeek$PeekSubscriber.onNext(FluxPeek.java:199)
	at reactor.core.publisher.FluxPeek$PeekSubscriber.onNext(FluxPeek.java:199)
	at reactor.core.publisher.FluxPeek$PeekSubscriber.onNext(FluxPeek.java:199)
	at reactor.core.publisher.FluxHandle$HandleSubscriber.onNext(FluxHandle.java:118)
	at reactor.core.publisher.FluxMap$MapConditionalSubscriber.onNext(FluxMap.java:220)
	at reactor.netty.channel.FluxReceive.drainReceiver(FluxReceive.java:265)
	at reactor.netty.channel.FluxReceive.onInboundNext(FluxReceive.java:371)
	at reactor.netty.channel.ChannelOperations.onInboundNext(ChannelOperations.java:381)
	at reactor.netty.channel.ChannelOperationsHandler.channelRead(ChannelOperationsHandler.java:94)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.io.IOException: Cannot run program "lib/faf-uid.exe": CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1142)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at java.base/java.lang.Runtime.exec(Runtime.java:591)
	at java.base/java.lang.Runtime.exec(Runtime.java:450)
	at com.faforever.client.os.OsUtils.execAndGetOutput(OsUtils.java:21)
	at com.faforever.client.legacy.WindowsUidService.generate(WindowsUidService.java:20)
	at com.faforever.client.remote.FafServerAccessorImpl.lambda$connectAndLogIn$4(FafServerAccessorImpl.java:126)
	at com.github.nocatch.NoCatch.noCatch(NoCatch.java:100)
	... 44 common frames omitted
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:483)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1109)
	... 51 common frames omitted
```

</p></details>

--- a/build.gradle
+++ b/build.gradle
@@ -144,6 +144,23 @@ configurations {
 
 bootRun {
     main = "com.faforever.client.Main"
+    jvmArgs = [
+            "-DnativeDir=build/resources/native",
+            "-Dprism.dirtyopts=false",
+            "-Dprism.forceGPU=true",
+            "-Xms128m",
+            "-Xmx712m",
+            "-XX:MinHeapFreeRatio=15",
+            "-XX:MaxHeapFreeRatio=33",
+            "-XX:+HeapDumpOnOutOfMemoryError",
+            "-XX:+UseStringDeduplication",
+            "-XX:ConcGCThreads=1",
+            "-XX:ParallelGCThreads=1",
+            "-XX:+UseG1GC",
+            "-javaagent:webview-patch/build/libs/webview-patch.jar",
+            "-Djava.net.preferIPv4Stack=true",
+            "-XX:TieredStopAtLevel=1"
+    ]
 }
 
 jar {

